nav2_dwb_controller/nav_2d_utils/include/nav_2d_utils/parameters.hpp

```diff
@@ -64,15 +64,6 @@ param_t searchAndGetParam(
   const nav2_util::LifecycleNode::SharedPtr & nh, const std::string & param_name,
   const param_t & default_value)
 {
-  // TODO(crdelsey): Handle searchParam
-  // std::string resolved_name;
-  // if (nh->searchParam(param_name, resolved_name))
-  // {
-  //   param_t value = 0;
-  //   nh->param(resolved_name, value, default_value);
-  //   return value;
-  // }
-  param_t value = 0;
```

Comment:
You deleted the variable? Doesn't that get referenced later?

Reply:
Of course, you're right... :) fixing, sorry for that